### DISCUSSION TITLE
How to disable caching in proxy(/absproxy/3000 or /proxy/3000)

### DISCUSSION THREAD

**Discussion by raychunghk** [coder/code-server#5655] [Q&A]
I am currently developing React front end with vs-code server.
On the front end proxy url:e.g:
http://code-serv/absproxy/3000/
I found that the result html rendered in browser is not updated when I change and save the js/html code.

But then when I open 
http://code-serv:3000
The updated code is in sync

So that is very likely the proxy is caching old version stuff.

Is it possible to turn it off?

**Comment by jsjoeio** [comment]
Hmm...my hunch would say that HMR isn't updating rather than the proxy caching your code.

Just to double-check, did you follow [these steps](https://coder.com/docs/code-server/latest/guide#proxying-to-create-a-react-app)?
